src/tools/status_tool.cpp

```diff
@@ -698,6 +700,40 @@ int StatusTool::PrintChunkserverStatus(bool checkLeftSize) {
     std::map<PoolIdType, std::vector<uint64_t>> poolChunkLeftSize;
     std::map<PoolIdType, std::vector<uint64_t>> poolWalSegmentLeftSize;
     std::vector<ChunkServerIdType> offlineCs;
+    bool useWalPool = false;
```

Comment:
This function is too long,  please split it into small function.

Reply:
> This function is too long, please split it into small function.

done